tests/m3u/test_translator.py

```diff
@@ -15,12 +15,15 @@
 data_dir = path_to_data_dir('')
 song1_path = path_to_data_dir('song1.mp3')
 song2_path = path_to_data_dir('song2.mp3')
+song3_path = path_to_data_dir(u'φοο.mp3')
```

Comment:
Because of the `unicode_literals` import, all strings in the file are already unicode literals. The u prefix is redundant.